diff --git a/modules/codec/arib/aribsub.c b/modules/codec/arib/aribsub.c
index 46272ea1758a..dcae414aa82c 100644
--- a/modules/codec/arib/aribsub.c
+++ b/modules/codec/arib/aribsub.c
@@ -27,6 +27,7 @@
 #include <vlc_common.h>
 #include <vlc_plugin.h>
 #include <vlc_codec.h>
+#include <vlc_strings.h>
 
 #ifdef HAVE_ARIBB24
  #include "substext.h"
@@ -34,6 +35,8 @@
  #include <aribb24/decoder.h>
 #endif
 
+#include "../../demux/mpeg/timestamps.h"
+
 //#define DEBUG_ARIBSUB 1
 
 /*****************************************************************************
@@ -177,10 +180,13 @@ static int Decode( decoder_t *p_dec, block_t *p_block )
     arib_decoder_t *p_decoder = arib_get_decoder( p_sys->p_arib_instance );
     if ( p_parser && p_decoder )
     {
-        arib_parse_pes( p_parser, p_block->p_buffer, p_block->i_buffer );
-        subpicture_t *p_spu = render( p_dec, p_parser, p_decoder, p_block );
+        char *b64 = vlc_b64_encode_binary(p_block->p_buffer, p_block->i_buffer);
+        msg_Dbg(p_dec, "arib_data [%s][%" PRId64 "]", b64, p_block->i_pts);
+        free(b64);
+        arib_parse_pes(p_parser, p_block->p_buffer, p_block->i_buffer);
+        /*subpicture_t *p_spu = render( p_dec, p_parser, p_decoder, p_block );
         if( p_spu != NULL )
-            decoder_QueueSub( p_dec, p_spu );
+            decoder_QueueSub( p_dec, p_spu );*/
     }
 
     block_Release( p_block );
