diff --git a/lib/core.c b/lib/core.c
index 5e8c614c83..34d26c7cf7 100644
--- a/lib/core.c
+++ b/lib/core.c
@@ -136,10 +136,11 @@ void libvlc_set_user_agent (libvlc_instance_t *p_i,
 {
     libvlc_int_t *p_libvlc = p_i->p_libvlc_int;
     char *str;
+    const char *plus = "(+https://github.com/vivid-lapin/vlc-miraktest)";
 
     var_SetString (p_libvlc, "user-agent", name);
     if ((http != NULL)
-     && (asprintf (&str, "%s LibVLC/"PACKAGE_VERSION, http) != -1))
+     && (asprintf (&str, "%s LibVLC/"PACKAGE_VERSION" %s", http, plus) != -1))
     {
         var_SetString (p_libvlc, "http-user-agent", str);
         free (str);
