diff --git a/modules/demux/mpeg/ts.c b/modules/demux/mpeg/ts.c
index 67abfdf44f..8b35e430ed 100644
--- a/modules/demux/mpeg/ts.c
+++ b/modules/demux/mpeg/ts.c
@@ -2202,6 +2202,7 @@ static void ProgramSetPCR( demux_t *p_demux, ts_pmt_t *p_pmt, mtime_t i_pcr )
     {
         p_pmt->pcr.i_first = i_pcr; // now seen
     }
+    msg_Dbg(p_demux, "i_pcr [%" PRId64 "][%" PRId64 "]", i_pcr, p_pmt->pcr.i_first);
 
     if ( p_sys->i_pmt_es )
     {
