diff --git a/src/libvlc.c b/src/libvlc.c
index 25f7c7d6d2..65f33988bf 100644
--- a/src/libvlc.c
+++ b/src/libvlc.c
@@ -269,10 +269,10 @@ int libvlc_InternalInit( libvlc_int_t *p_libvlc, int i_argc,
      * (in place of VLC main()) *here*. */
     var_Create( p_libvlc, "user-agent", VLC_VAR_STRING );
     var_SetString( p_libvlc, "user-agent",
-                   "VLC media player (LibVLC "VERSION")" );
+                   "VLC media player (LibVLC "VERSION") vlc-miraktest (+https://github.com/vivid-lapin/vlc-miraktest)" );
     var_Create( p_libvlc, "http-user-agent", VLC_VAR_STRING );
     var_SetString( p_libvlc, "http-user-agent",
-                   "VLC/"PACKAGE_VERSION" LibVLC/"PACKAGE_VERSION );
+                   "VLC/"PACKAGE_VERSION" LibVLC/"PACKAGE_VERSION" vlc-miraktest (+https://github.com/vivid-lapin/vlc-miraktest)" );
     var_Create( p_libvlc, "app-icon-name", VLC_VAR_STRING );
     var_SetString( p_libvlc, "app-icon-name", PACKAGE_NAME );
     var_Create( p_libvlc, "app-id", VLC_VAR_STRING );
